### PR TITLE
fdesetup: add page

### PR DESCRIPTION
**Pull request by emilshouse**
- [x] The page (if new), does not already exist in the repo.
- [x ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x ] The page has 8 or fewer examples.
- [x ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/osx/fdesetup.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- List current FileVault enabled users:
+
+`sudo fdesetup list`
+
+- Get current FileVault status:
+
+`sudo fdesetup status`
+
+- Add fileVault enabled user:
```
Comment (zdroid) [inline pages/osx/fdesetup.md:13]:
```suggestion
- Add FileVault enabled user:
```

File: pages/osx/fdesetup.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# fdesetup
+
+> Set and retrieve FileVault related information. Sudo required for most commands.
```
Comment (zdroid) [inline pages/osx/fdesetup.md:3]:
```suggestion
> Set and retrieve FileVault related information.
> Sudo required for most commands.
```
Should be in this format. Maybe even remove or rephrase the second part.

File: pages/osx/fdesetup.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# fdesetup
+
+> Set and retrieve FileVault related information.
+> Sudo required for most commands.
```
Comment (andrik) [inline pages/osx/fdesetup.md:4]:
```suggestion
```
Since **sudo** is required for all examples, I think we should omit this line

Comment (zdroid) [reply pages/osx/fdesetup.md:4]: Oh right. `sudo` is already written in all the examples.

Comment (emilshouse) [reply pages/osx/fdesetup.md:4]: Sounds good. I will go ahead and remove that line.

**Comment by andrik** [comment]
Hi @emilshouse 

Thank you for your contribution

**Comment by emilshouse** [comment]
@andrik @zdroid  Happy to help! Thanks for your input also.

**Pull request merged**